### ISSUE TITLE
The oe_theme JS is correctly loaded when the website has only one language

### ISSUE THREAD

**Issue by amd-miri** [openeuropa/oe_theme#414]
Hello,
we found an issue in the oe_theme. We implemented the in-page-navigation component twig in our website and then we realised that the related JS function are not ran correctly. After checking the browser console log, we found the following error :

TypeError: this.languageSelector is null

Then I added a second language to our website and everything worked again.
The problem is that our website is only in English and we don't want any other language.

![image](https://user-images.githubusercontent.com/16777387/73279779-eac42880-41ed-11ea-946e-bda18593877d.png)

Regards,
Ahmad

**Comment by ademarco** [comment]
That might be an issue with the ECL preset, could you please expand the error message so I get a look at the call stack?

**Comment by ademarco** [comment]
Yep, I've checked with the ECL team and it's an issue on their side. They are on it, once they release a fix we will include the new library version in the theme and release, I'll let you know when we will do that.

**Comment by amd-miri** [comment]
Thanks @ademarco for the update :)
